CMakeLists.txt

```diff
@@ -272,7 +272,13 @@ INCLUDE(${KOKKOS_SRC_PATH}/cmake/kokkos_install.cmake)
 # executables also need nvcc_wrapper.  Thus, we need to install it.
 # If the argument of DESTINATION is a relative path, CMake computes it
 # as relative to ${CMAKE_INSTALL_PATH}.
-INSTALL(PROGRAMS ${CMAKE_CURRENT_SOURCE_DIR}/bin/nvcc_wrapper DESTINATION ${CMAKE_INSTALL_BINDIR})
+# Also install other utility scripts
+SET(BIN_SCRIPTS
+  nvcc_wrapper
+  hpcbind)
+FOREACH(SCRIPT ${BIN_SCRIPTS})
+  INSTALL(PROGRAMS ${CMAKE_CURRENT_SOURCE_DIR}/bin/${SCRIPT} DESTINATION ${CMAKE_INSTALL_BINDIR})
+ENDFOREACH()
```

Comment:
Any reason why you preferred a loop even though the `install` command supports 
```CMake
INSTALL(
  PROGRAMS
    bin/nvcc_wrapper
    bin/hpcbind
  DESTINATION ${CMAKE_INSTALL_BINDIR})
```

Reply:
Because I was trying to fit in work between my child's naps. Let us pretend this didn't happen and never speak of this again.